bsp_diff/common/vendor/intel/external/project-celadon/cloud-streaming/0001-Add-entry-point-script-for-aic-cloud-env.patch

```diff
@@ -167,10 +170,15 @@ index 0000000..19edf25
 +
 +  local renderfps_enc_enable=${ENV_RENDERFPS_ENC_ENABLE:-"false"}
 +  local hevc_enable=${HEVC_ENABLE:-"false"}
++  local av1_enable=${AV1_ENABLE:-"false"}
```

Comment:
That's bad options style. Problem with this style of options is that they conflict with each other. User has a chance to type in: `-hevc -av1` and one of them will take preference.